examples/resources/spectrocloud_cluster_azure/resource.tf

```diff
@@ -35,7 +34,6 @@ resource "spectrocloud_cluster_azure" "cluster" {
   }
 
   machine_pool {
-    is_system_node_pool = true
```

Comment:
please keep it in the example.

Reply:
okay

Reply:
@SivaanandM let's also set a default value in golang. I assume it will be false.

Reply:
updated